.env

```diff
@@ -14,6 +14,94 @@ HOST_URL=${SERVER_SCHEME}://${HOST_NAME}
 TIMEZONE=UTC
 GITPOD_ENV=false
 ENABLE_SSO=false
+
+# Bahmni EMR
+
+RUN_WITH_BAHMNI_EMR=false
+BAHMNI_TEST_ORDER_TYPE_UUID=f8ae333e-9d1a-423e-a6a8-c3a0687ebcf2
+
+# Mail Config Properties Environment Variables
+MAIL_TRANSPORT_PROTOCOL=smtps
+MAIL_SMTP_AUTH=true
+MAIL_SMTP_STARTTLS_ENABLE=true
+MAIL_SMTP_SSL_ENABLE=true
+MAIL_DEBUG=false
+SEND_MAIL=false
+MAIL_FROM=
+MAIL_SMTP_HOST=
+MAIL_SMTP_PORT=
+MAIL_USER=
+MAIL_PASSWORD=
+
+# Bahmni Config Environment Variables
+BAHMNI_CONFIG_IMAGE_TAG=1.0.0
+BAHMNI_CONFIG_VOLUME=bahmni-config
+BAHMNI_CONFIG_OVERRIDE_PATH=
+
+# Bahmni - OpenMRS Environment Variables
+OPENMRS_HOST=openmrs
+OPENMRS_PORT=8080
+OPENMRS_IMAGE_TAG=1.1.1
+OPENMRS_DB_IMAGE_NAME=bahmni/openmrs-db:1.0.0-standard
+OPENMRS_DB_NAME=openmrs
+OPENMRS_DB_HOST=openmrsdb
+OPENMRS_DB_USERNAME=openmrs-user
+OPENMRS_DB_PASSWORD=password
```

Comment:
Most of these env variables already exists for O3 (OpenMRS). To differentiate from OpenMRS env vars, maybe we can add the prefix "BAHMNI_".